### PR TITLE
[Issue 5134][docs] Add replicated subscription feature

### LINKED ISSUE #5134: [Documentation] Add Geo Replicated subscriptions documentation in website
In [PIP-33](https://github.com/apache/pulsar/wiki/PIP-33%3A-Replicated-subscriptions), Geo Replicated subscriptions, feature is added, but documentation is not updated.
related PRs: #4396 #4299 #4340 #4354

This issue tries to track the documentation work.

### PR DESCRIPTION
**Pull request by Jennifer88huang**
Fixes #5134 

### Motivation
The geo Replicated subscriptions feature is added, but documentation is not updated.

### Modifications
Add replicated subscription docs based on https://github.com/apache/pulsar/wiki/PIP-33:-Replicated-subscriptions

### CODE REVIEW

**Comment by Jennifer88huang** [comment]
@tuteng @zymap Could you please help review? Thank you.
There are things to be confirmed, see discussion in https://github.com/apache/pulsar/issues/5134

**Comment by Jennifer88huang** [comment]
Shall we propagate the following info to users? If those are set by default, and user need not to configure or know the logic, we'll not add those info in user docs.
- snapshot solution
- maker messages
- construct cursor snapshot
- store snapshot
- update subscription in remote regions

File: site2/docs/administration-geo.md [hunk 1]
```diff
@@ -126,3 +126,32 @@ For geo-replication topics, each region uses a fault-tolerant mechanism to decid
 You can explicitly disable topic garbage collection by setting `brokerDeleteInactiveTopicsEnabled` to `false` in your [broker configuration](reference-configuration.md#broker).
 
 To delete a geo-replication topic, close all producers and consumers on the topic, and delete all of its local subscriptions in every replication cluster. When Pulsar determines that no valid subscription for the topic remains across the system, it will garbage collect the topic.
+
+## Replicated subscriptions
+
+Pulsar supports replicated subscriptions, so you can keep subscription state in sync, within a sub-second timeframe, in the context of a topic that is being asynchronously replicated across multiple geographical regions.
+
+In case of failover, a consumer can restart consuming from the failure point.
```
Comment (merlimat) [inline site2/docs/administration-geo.md]: It should mention that the consumer can restart consuming from a different cluster

Comment (Jennifer88huang) [reply site2/docs/administration-geo.md]: Got it, thanks for your information.

**Comment by Jennifer88huang** [comment]
@merlimat @tuteng @zymap @jiazhai Could you please help review it? Thank you.

File: site2/docs/administration-geo.md [hunk 2]
```diff
@@ -125,4 +125,33 @@ For geo-replication topics, each region uses a fault-tolerant mechanism to decid
...
+
+## Replicated subscriptions
+
+Pulsar supports replicated subscriptions, so you can keep subscription state in sync, within a sub-second timeframe, in the context of a topic that is being asynchronously replicated across multiple geographical regions.
+
+In case of failover, a consumer can restart consuming from the failure point in a different cluster. 
+
+### Enable replicated subscription
+
+Replicated subscription is enabled/disabled by default. You can enable replicated subscription when creating a consumer.
```
Comment (zymap) [inline site2/docs/administration-geo.md]: disable by default.

Comment (Jennifer88huang) [reply site2/docs/administration-geo.md]: @zymap Thank you very much for your advice. Updated.

**Comment by tuteng** [comment]
LGTM

**Pull request merged**